Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM defradigital/node-development:22.4.0-alpine3.19 as dev
+FROM defradigital/node-development:2.3.0-node22.4.0 as dev
 
 WORKDIR /usr/src/app
 COPY package*.json ./
 RUN npm ci --only=development
 COPY . .
 
-RUN npm run build:dev
```

Comment:
I think we should have a chat where we are going with this as we're diverging from https://defra.github.io/ffc-development-guide/development-patterns/containers/